dotCMS/src/main/webapp/html/js/dotcms/dijit/tree/HostFolderTreeReadStoreModel.js

```diff
@@ -115,6 +115,17 @@ dojo.declare("dotcms.dijit.tree.HostFolderTreeReadStoreModel", null, {
 		}
 	},
 
+	getFullPath: function(item) {
+		let fullPath = "";
+		const Host = item.fullPath.split(":")[0];
+		if (item.type === "host") {
+			fullPath = Host
+		} else {
+			fullPath = `${Host}${item.folderPath}`
+		}
+		return fullPath
```

Comment:
You can remove the if condition and the fullPath variable by using a ternary operator and you can use destructuring to take the first index of an array
```suggestion
                const [ host ] = item.fullPath.split(":");
		return item.type === "host" ? Host : `${Host}${item.folderPath}`
```
Also I recommend to change `Host` for `host` to maintain the standard

Reply:
Done!